CogNative/main.py

```diff
@@ -12,6 +11,7 @@
 from .backend.modules.translation import translation
 from .backend.modules.STT import STT
 from .backend.modules.utils import mp3_to_wav
+from .backend.modules.text_split import split_text
```

Comment:
```suggestion
from .backend.modules.utils import mp3_to_wav, split_text
```

I think you should put split_text in utils.py. I think it would be easier if we just kept all of these support functions in the same file.